lib/foreman_maintain/cli/base.rb

```diff
@@ -101,7 +103,9 @@ def self.interactive_option
         delete_duplicate_assumeyes_if_any
 
         option ['-y', '--assumeyes'], :flag,
-               'Automatically answer yes for all questions'
+               'Automatically answer yes for all questions' do |assume|
+          ForemanMaintain.reporter.assumeyes(assume)
```

Comment:
```
# ./bin/foreman-maintain advanced procedure run hammer-setup -y
undefined method `assumeyes' for #<ForemanMaintain::Reporter::CLIReporter:0x000000024ef3e8>
```

```
# ./bin/foreman-maintain health check -y
undefined method `assumeyes' for #<ForemanMaintain::Reporter::CLIReporter:0x00000001a283d8>
```

Reply:
Fixed, this was last minute change suggested by rubocop :)